src/pages/WorkspaceSwitcherPage/index.tsx

```diff
@@ -29,12 +29,12 @@ function WorkspaceSwitcherPage() {
     const {translate} = useLocalize();
     const {activeWorkspaceID} = useActiveWorkspace();
 
-    const [reports] = useOnyx(ONYXKEYS.COLLECTION.REPORT);
-    const [reportActions] = useOnyx(ONYXKEYS.COLLECTION.REPORT_ACTIONS);
-    const [policies, fetchStatus] = useOnyx(ONYXKEYS.COLLECTION.POLICY);
-    const [currentUserLogin] = useOnyx(ONYXKEYS.SESSION, {selector: (session) => session?.email});
-    const [isLoadingApp] = useOnyx(ONYXKEYS.IS_LOADING_APP);
-    const brickRoadsForPolicies = useMemo(() => getWorkspacesBrickRoads(reports, policies, reportActions), [reports, policies, reportActions]);
+    const [reports] = useOnyx(ONYXKEYS.COLLECTION.REPORT, {canBeMissing: true});
+    const [reportActions] = useOnyx(ONYXKEYS.COLLECTION.REPORT_ACTIONS, {canBeMissing: true});
+    const [policies, fetchStatus] = useOnyx(ONYXKEYS.COLLECTION.POLICY, {canBeMissing: true});
+    const [currentUserLogin] = useOnyx(ONYXKEYS.SESSION, {selector: (session) => session?.email, canBeMissing: true});
+    const [isLoadingApp] = useOnyx(ONYXKEYS.IS_LOADING_APP, {canBeMissing: true});
```

Comment:
can these be missing really?

Reply:
This property is here for a while now but I still find it super confusing. Out of curiosity I just checked the codebase and this key is used 4 times with `false` and 11 times with `true` and I can't really say what's the difference. The JSDoc description of this param is not clear to me and based on these numbers I think it might be problematic for more people. Can we do something to improve it? cc @fabioh8010